pages/common/jenv.md

```diff
@@ -0,0 +1,28 @@
+# jenv
+
+> JEnv is a command line tool to help you forget how to set the "JAVA_HOME" environment variable.
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
> Command line tool to manage the "JAVA_HOME" environment variable.
```